src/components/Counter.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+
+function Counter({ fruitName, count, increment, decrement }) {
+
+    return (
+        <>
+        <div className="counter">
+            <h2>{fruitName}</h2>
+            <button onClick={decrement}>-</button>
```

Comment:
Het is netjes om de button disabled te maken als je count op 0 staat. Op die manier had je niet een speciale decrement functie hoeven hebben die het daarin afvangt.